### DISCUSSION TITLE
Why NX's version bumps so fast?

### DISCUSSION THREAD

**Discussion by turnerguo** [nrwl/nx#19944] [General]
I see [Nx 16 Roadmap](https://github.com/nrwl/nx/discussions/12836) still their, but NX 17 is already came out, and I didn't see any BREAKING CHANGE in NX 17 but why bump the version so fast?
How do we planned the versioning? it seems does not follow the semver?

**Comment by ahernd2** [comment]
If I had to guess, it was because of the release of angular 17, which would be a breaking change
